src/partition/tests/ProvidedPartitionTest.cpp

```diff
@@ -360,38 +360,30 @@ BOOST_AUTO_TEST_CASE(TestCompareBoundingBoxes2D)
     BOOST_TEST(context.isNamed("NASTIN"));
 
     mesh::Mesh::BoundingBoxMap receivedGlobalBB;
-    mesh::Mesh::BoundingBox    localBB;
+    mesh::BoundingBox    localBB{dimensions};
+
+    mesh::Mesh::BoundingBoxMap compareBB;
+    compareBB[0] = mesh::BoundingBox::createFromData({-1, 5, 0, 3});
+    compareBB[1] = mesh::BoundingBox::createFromData({0, 1, 3.5, 4.5});
+    compareBB[2] = mesh::BoundingBox::createFromData({2.5, 4.5, 5.5, 7.0});
```

Comment:
Since I removed the constructor which takes `std::vector` and use only `createFromData() based on your previous feedback, I think it can stay as it is.